forte/models/da_rl/generator_trainer.py

```diff
@@ -0,0 +1,398 @@
+# Copyright 2020 The Forte Authors. All Rights Reserved.
```

Comment:
I don't think the name of this file should be `generator_trainer` though. First this class mainly contains the wrapper, secondly, `generator` and `trainer` are both very general names, we don't know what to expect from such a file.